homeassistant/components/unifiprotect/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "UniFi Protect",
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/unifiprotect",
-  "requirements": ["pyunifiprotect==4.1.9", "unifi-discovery==1.1.6"],
+  "requirements": ["pyunifiprotect==4.2.0", "unifi-discovery==1.1.5"],
```

Comment:
Looks like this is an accidental revert?

Reply:
Whoops. Nice catch.